src/java/org/apache/cassandra/config/Config.java

```diff
@@ -742,6 +742,8 @@ public static void setClientMode(boolean clientMode)
     public volatile Set<String> table_properties_disallowed = Collections.emptySet();
     public volatile boolean user_timestamps_enabled = true;
     public volatile boolean read_before_write_list_operations_enabled = true;
+    public volatile DurationSpec streaming_state_expires = DurationSpec.inDays(3);
```

Comment:
even though this is dynamic (could be updated and changes behavior), holding off adding JMX with the hope that CASSANDRA-15254 solves this generically...